solutions/devsprint-riccardo-1/OnboardingChallenge/Helpers/SwiftUIPreView.swift

```diff
@@ -36,3 +36,43 @@ import Foundation
     func updateUIView(_: V, context _: Context) {}
   }
 #endif
+
+#if canImport(SwiftUI) && DEBUG
+import SwiftUI
+
+struct UIViewPreview<View: UIView>: UIViewRepresentable {
+    let view: View
+    
+    init(_ builder: @escaping () -> View) {
+        view = builder()
+    }
+    
+    func makeUIView(context: Context) -> View {
+        return view
+    }
+    
+    func updateUIView(_ uiView: View, context: Context) {
+        uiView.setContentHuggingPriority(.defaultHigh, for: .horizontal)
+        uiView.setContentHuggingPriority(.defaultHigh, for: .vertical)
+    }
+}
+
+@available(iOS 13.0, *)
+extension UIViewController {
```

Comment:
An important concept when working in a team is that every pr should have a single responsibility. Improving the SwiftUIPreview template was not the responsibility of this PR, which should be focused on implementing the button view. 
This welcomed addendum should go in a separate PR, where we address the possibility to make a ViewController representable in a SwiftUI preview.